src/runtime/container.cc

```diff
@@ -21,13 +21,19 @@
  * \file src/runtime/container.cc
  * \brief Implementations of common containers.
  */
+#include <tvm/node/container.h>
```

Comment:
Shouldn't we avoid including node into runtime?

Reply:
Good catch @FrozenGene can we remove the use of Map and use std::unordered_map for now?